src/App.css

```diff
@@ -28,3 +28,131 @@ html {
 	justify-content: center;
 	font-family: 'Share Tech Mono', monospace;
 }
+
+.dialog-container {
+	display: flex;
+	position: fixed;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+}
+
+.dialog-container[aria-hidden='true'] {
+	display: none;
+}
+
+.dialog-overlay {
+	position: fixed;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	background-color: rgba(43, 46, 56, 0.9);
+	animation: fade-in 200ms both;
+}
+
+.dialog-content {
+	font-family: 'Share Tech Mono', monospace;
+	background-color: var(--yellow);
+	margin: auto;
+	z-index: 2;
+	position: relative;
+	animation: fade-in 400ms 200ms both, slide-up 400ms 200ms both;
+	padding: 1em;
+	max-width: 90%;
+	width: 600px;
+	border-radius: 2px;
+	color: var(--blue);
+}
+
+@media screen and (min-width: 700px) {
+	.dialog-content {
+		padding: 2em;
+	}
+}
+
+@keyframes fade-in {
+	from {
+		opacity: 0;
+	}
+}
+
+@keyframes slide-up {
+	from {
+		transform: translateY(10%);
+	}
+}
+
+.dialog-title {
+	margin: 0;
+	font-size: 1.25em;
+}
+
+.dialog-close {
+	position: absolute;
+	top: 0.5em;
+	right: 0.5em;
+	border: 0;
+	padding: 0;
+	background-color: transparent;
+	font-weight: bold;
+	font-size: 1.25em;
+	width: 1.2em;
+	height: 1.2em;
+	text-align: center;
+	cursor: pointer;
+	transition: 0.15s;
+}
+
+@media screen and (min-width: 700px) {
+	.dialog-close {
+		top: 1em;
+		right: 1em;
+	}
+}
+
+* {
+	box-sizing: border-box;
+}
+
+.delete-confirm-button {
+	background-color: var(--blue);
+	color: var(--yellow);
+	border: 0;
+	font-family: inherit;
+	font-size: inherit;
+	padding: 8px 15px;
+	cursor: pointer;
+	transition: 0.15s;
+	display: inline-block;
+	border: 1px solid var(--blue);
+	width: 5em;
+}
+
+.delete-confirm-button:hover,
+.delete-confirm-button:active {
```

Comment:
I really love the hover color changes of the buttons on the modal! This might sound nit-picky so its not a big deal if we don't make this change... but.. I was slightly confused by the hover at first. When the modal open... the confirm or 'Ok' button is Blue with white font. The 'Cancel' button is white with blue font. Then, when you hover over the 'Ok' button, it transitions and matches the 'Cancel' button. It almost looks as though the 'Ok' button was DE-selected. If you hover over the 'Cancel' button, it transitions to Blue with White font, matching the 'Ok' button. It appears then as though both buttons are not selected.

Idk if that even makes sense....? I'm sorry if not.

Reply:
Ohhh I agree! My suggested edit: 

In the `.delete-confirm-button` styling, change: 
	    `background-color: var(--white);`
	    `color: var(--blue);`

Then, combine the `::hover` and `::active` code, so that both buttons have the same behavior: 

`.delete-confirm-button:hover,`
`.delete-confirm-button:active,`
`.delete-cancel-button:hover,`
`.delete-cancel-button:active {`
	`background-color: var(--blue);`
	`color: var(--yellow);`
`}`

Reply:
I saw this use of different colors for call-to-action in some websites and decided to do something like this. I'll change the colors if they are causing the confusion.